### DISCUSSION TITLE
缓存的接口如果成功返回了数据, 成功的拦截器不会响应?

### DISCUSSION THREAD

**Discussion by eivmosn** [alovajs/alova#204] [General]
拦截器添加了接口等待的 loadingbar 动画,  只有成功返回数据才会被停止, 但是被缓存的接口 好像不会执行 成功的响应拦截器哦 导致动画没法关闭了

**Comment by JOU-amjs** [comment]
@eivmosn 确实，如果命中了缓存的话，`responded`回调不会再被触发了，当时它的设计就是网络响应后的回调钩子。
不过我这边有一个取巧的办法，你可以使用[全局的缓存命中钩子cacheLogger](https://alova.js.org/zh-CN/tutorial/next-step/cache-logger)来停止loadingbar动画，它会在命中缓存时触发，同时配合`responded`可以达到你想要的效果
